docs: clarify the behavior of includes (#1532)
pdm-project/pdm: docs/docs/pyproject/build.md

]
```

!!! note

> 	When using `includes` the package itself is not included by default. You have to add it manually as well.

Reviewer: ```suggestion
	When using `includes` the default includes will be overridden. You have to add the package paths manually.
```
